includes/admin-intro.php

```diff
@@ -0,0 +1,416 @@
+<?php
+
+if (!defined('ABSPATH')) {
+    die;
+}
+
+define('WPRSS_INTRO_PAGE_SLUG', 'wpra-intro');
+define('WPRSS_FIRST_ACTIVATION_OPTION', 'wprss_first_activation_time');
+define('WPRSS_PREV_WELCOME_SCREEN_OPTION', 'wprss_pwsv');
+define('WPRSS_INTRO_DONE_OPTION', 'wprss_did_intro');
+define('WPRSS_INTRO_STEP_OPTION', 'wprss_intro_step');
+define('WPRSS_INTRO_NONCE_NAME', 'wprss_intro_nonce');
+define('WPRSS_INTRO_STEP_POST_PARAM', 'wprss_intro_step');
+define('WPRSS_INTRO_FEED_URL_PARAM', 'wprss_intro_feed_url');
+define('WPRSS_INTRO_SHORTCODE_PAGE_OPTION', 'wprss_intro_shortcode_page');
+define('WPRSS_INTRO_SHORTCODE_PAGE_PREVIEW_PARAM', 'wprss_preview_shortcode_page');
+
+/**
+ * Registers the introduction page.
+ *
+ * @since [*next-version*]
+ */
+add_action('admin_menu', function () {
+    add_submenu_page(
+        null,
+        __('Welcome to WP RSS Aggregator'),
+        __('Welcome to WP RSS Aggregator'),
+        'read',
+        WPRSS_INTRO_PAGE_SLUG,
+        'wpra_render_intro_page'
+    );
+});
+
+/**
+ * Renders the intro page.
+ *
+ * @since [*next-version*]
+ *
+ * @throws Twig_Error_Loader
+ * @throws Twig_Error_Runtime
+ * @throws Twig_Error_Syntax
+ */
+function wpra_render_intro_page()
+{
+    wp_enqueue_script('intro-wizard', WPRSS_JS . 'intro.min.js', [], '0.1', true);
+    wp_enqueue_style('intro-wizard', WPRSS_CSS . 'intro.min.css');
+
+    $nonce = wp_create_nonce(WPRSS_INTRO_NONCE_NAME);
+    wp_localize_script('intro-wizard', 'wprssWizardConfig', [
+        'previewUrl' => admin_url('admin.php?wprss_preview_shortcode_page=1&nonce=' . $nonce),
+        'feedEndpoint' => [
+            'url' => admin_url('admin-ajax.php'),
+            'defaultPayload' => [
+                'action' => 'wprss_create_intro_feed',
+                'nonce' => $nonce,
+            ],
+        ],
+        'saveStepEnpoint' => [
```

Comment:
@romalytvynenko Is this a typo? 😅

Reply:
@mecha, yeah, this is typo. I'll fix in the next commit 👌